macos/package_files/3.9.3/preinstall.sh

```diff
@@ -17,6 +17,17 @@ else
     launchctl setenv WAZUH_PKG_UPGRADE true
 fi
 
+if [ $(launchctl getenv WAZUH_PKG_UPGRADE) = true ]; then
+    mkdir -p ${DIR}/config_files/
+    cp -r ${DIR}/etc/{ossec.conf,client.keys,local_internal_options.conf,shared} ${DIR}/config_files/
+fi
+
+if [ $(launchctl getenv WAZUH_PKG_UPGRADE) = true ]; then
+    if [ `pkgutil --pkgs | grep -i wazuh-agent-etc` > /dev/null 2>&1 ]; then
```

Comment:
Here you are using legacy notation. Change this line as I said in this comment https://github.com/wazuh/wazuh-packages/pull/221#discussion_r299574968

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/70bb6f0df4fa55ec15c4af233aa22793c1190571